### ISSUE TITLE
Template not found after 3.2.0

### ISSUE THREAD

**Issue by zachary-highman** [Snowflake-Labs/schemachange#85]
No issues with 3.1.0. We supply a JSON object for our variables with --vars and do not use a config file. Below is our call to schemachange:

`schemachange -f MORE2/ -a ${SF_ACCOUNT} -u ${SF_USERNAME} -r ${SF_ROLE} -w ${SF_WAREHOUSE} -d ${SF_DATABASE} -c ${SF_DATABASE}.SCHEMACHANGE.CHANGE_HISTORY --create-change-history-table --vars $VARS -v`

Here is the error:

```
2021-11-02T14:39:11.1821401Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\runpy.py", line 193, in _run_module_as_main
2021-11-02T14:39:11.1825161Z     "__main__", mod_spec)
2021-11-02T14:39:11.1826123Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\runpy.py", line 85, in _run_code
2021-11-02T14:39:11.1826631Z     exec(code, run_globals)
2021-11-02T14:39:11.1827900Z   File "C:\hostedtoolcache\windows\Python\3.7.9\x64\Scripts\schemachange.exe\__main__.py", line 7, in <module>
2021-11-02T14:39:11.1828807Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\schemachange\cli.py", line 606, in main
2021-11-02T14:39:11.1829929Z     deploy_command(config)
2021-11-02T14:39:11.1830572Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\schemachange\cli.py", line 147, in deploy_command
2021-11-02T14:39:11.1831200Z     content = jinja_processor.render(jinja_processor.relpath(script['script_full_path']), config['vars'], config['verbose'])
2021-11-02T14:39:11.1832008Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\schemachange\cli.py", line 53, in render
2021-11-02T14:39:11.1832533Z     template = self.__environment.get_template(script)
2021-11-02T14:39:11.1833048Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\jinja2\environment.py", line 997, in get_template
2021-11-02T14:39:11.1833888Z     return self._load_template(name, globals)
2021-11-02T14:39:11.1834363Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\jinja2\environment.py", line 958, in _load_template
2021-11-02T14:39:11.1834887Z     template = self.loader.load(self, name, self.make_globals(globals))
2021-11-02T14:39:11.1835701Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\jinja2\loaders.py", line 125, in load
2021-11-02T14:39:11.1836426Z     source, filename, uptodate = self.get_source(environment, name)
2021-11-02T14:39:11.1837121Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\jinja2\loaders.py", line 194, in get_source
2021-11-02T14:39:11.1837599Z     pieces = split_template_path(template)
2021-11-02T14:39:11.1838103Z   File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\jinja2\loaders.py", line 35, in split_template_path
2021-11-02T14:39:11.1838569Z     raise TemplateNotFound(template)
2021-11-02T14:39:11.1839207Z jinja2.exceptions.TemplateNotFound: SQL\V2.0.0__ADHOC_SCRIPT.sql
```

This is working still with 3.1.0. Is there something missing in our CLI call?

**Comment by saperry** [comment]
Hi @zachary-highman, nothing wrong with your CLI call, you have found a bug. Version 3.2.1 has now been released which contains a fix for your scenario. It would be great if you could confirm that the new version fixes the issue.

**Comment by sfc-gh-jhansen** [comment]
Yes @zachary-highman, this should be fixed now with versions 3.2.1 and greater. Please let us know if you're still having issues.
